Source/Core/DolphinQt/Config/InfoWidget.cpp

```diff
@@ -121,6 +126,19 @@ QGroupBox* InfoWidget::CreateGameDetails()
   if (!m_game.GetApploaderDate().empty())
     layout->addRow(tr("Apploader Date:"), CreateValueDisplay(m_game.GetApploaderDate()));
 
+  if (m_volume)
+  {
+    layout->addRow(tr("Region:"), CreateValueDisplay(fmt::format("{}", m_volume->GetRegion())));
```

Comment:
NBD but is there a way to make the region into text?  We could do:  `<region text> (<region code>)` which would be more understandable than just the number

Reply:
I guess "country" is good enough...?

Reply:
Yeah I guess the country field is good enough since it's the result of TypicalCountryForRegion(region)... I'll drop the same commit